adaptors/adaptors.pro

```diff
@@ -54,6 +54,20 @@ SUBDIRS += steaccelerometeradaptor
 SUBDIRS += mpu6050accelerometer
 }
 
+config_hybris {
```

Comment:
makes me wonder whether we should only use the hybris config instead of autohybris. And we don't provide it on the qemu targets

Reply:
This is what I did initially, but it caused different build issues related to undefined data types. I think this line is the erroneous https://github.com/MagneFire/sensorfw/blob/gatesgarth_v2/sensorfw.pro#L62 instead of adding folders it replaces the previously set datatypes path https://github.com/MagneFire/sensorfw/blob/gatesgarth_v2/sensorfw.pro#L18

I'll look into this some more maybe there is a more appropriate fix rather than partially undoing a commit (https://github.com/MagneFire/sensorfw/commit/b43d36e8a5796361bbbc9d541d4127fd897bd1c3)